### ISSUE TITLE
header json.h missing from Vega-Strike-Engine-Source/engine/src/configuration/

### ISSUE THREAD

**Issue by stamasd** [vegastrike/Vega-Strike-Engine-Source#937]
Attempting to compile the code freshly pulled from the repo as of today 12/26/24 results in:

[  8%] Building CXX object CMakeFiles/vegastrike-engine_com.dir/src/configuration/configuration.cpp.o
/home/user/games/vegastrike/Vega-Strike-Engine-Source/engine/src/configuration/configuration.cpp:28:10: fatal error: json.h: No such file or directory
   28 | #include "json.h"
      |          ^~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/vegastrike-engine_com.dir/build.make:104: CMakeFiles/vegastrike-engine_com.dir/src/configuration/configuration.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:273: CMakeFiles/vegastrike-engine_com.dir/all] Error 2
make: *** [Makefile:166: all] Error 2

And indeed there is no file "json.h" in Vega-Strike-Engine-Source/engine/src/configuration/

**Comment by BenjamenMeyer** [comment]
Yes I found this as well; not sure how it got past our CI builds. #938 should resolve it.
I have some local build issues so if you would please test it that would be appreciated.

**Comment by BenjamenMeyer** [comment]
@evertvorster could you help verify #938 as a fix?

**Comment by evertvorster** [comment]
Hi there, Benjamin. 

Unfortunately, no. 
Please see #941

Seems that something on my system got updated, and now I am unable to build branches that I know have worked before.
